### ISSUE TITLE
KMeans n_clusters bad type check

### ISSUE THREAD

**Issue by mdymczyk** [h2oai/h2o4gpu#281]
Got while running DAI on homesite data:

`Argument "n_clusters" should be an integer | type | type, got int64 23`

To repro:
```
from h2o4gpu import KMeans
import numpy as np
k = np.int64(2)
KMeans(n_clusters=k)
```

**Comment by pseudotensor** [comment]
Is that from your code?

**Comment by mdymczyk** [comment]
@pseudotensor the type checker thing is failing. My guess is it will fail anywhere where we use `assert_is_type` and allow `numpy` types.

**Comment by mdymczyk** [comment]
ok so the issue is we're using `type(np.int32)` etc. in `assert_is_type` while we should simply use `np.int32`.

**Comment by mdymczyk** [comment]
Fixed in #285
